sdk/openai/azure-ai-openai-assistants/src/main/java/com/azure/ai/openai/assistants/models/VectorStoreChunkingStrategyResponseType.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.assistants.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+/**
+ * Type of chunking strategy.
+ */
+public final class VectorStoreChunkingStrategyResponseType
```

Comment:
Why do we have different enum values for response and request chunking strategy types?

Reply:
The request type is an expandable enum with values `auto` and `static` and the response is `auto` and `other`. Despite the overlap with `auto` I think it's best to maintain these enums separate.